homeassistant/components/zha/binary_sensor.py

```diff
@@ -41,6 +42,7 @@
     OPENING: OPENING,
     ZONE: get_ias_device_class,
     OCCUPANCY: OCCUPANCY,
+    ACCELERATION: ACCELERATION,
```

Comment:
should be ACCELERATION: 'moving'... that said, import the constant for moving and use that.

Reply:
Done